plugins/spu2-x/src/SndOut.cpp

```diff
@@ -390,8 +390,11 @@ void SndBuffer::Init()
 
     // clear buffers!
     // Fixes loopy sounds on emu resets.
-    memset(sndTempBuffer, 0, sizeof(StereoOut32) * SndOutPacketSize);
-    memset(sndTempBuffer16, 0, sizeof(StereoOut16) * SndOutPacketSize);
+
+    // Per turtleli, the initialization here should no longer be necessary.
+    // Leaving the code commented out in case the issue above returns.
+    // std::fill_n(sndTempBuffer, SndOutPacketSize, StereoOut32{});
+    // std::fill_n(sndTempBuffer16, SndOutPacketSize, StereoOut16{});
```

Comment:
I'd suggest deleting this instead of leaving an updated commented out version. You can always use a debugger to convince yourself that the buffers are zero initialised at this point.